test/functional/services/dashboard/panel_settings.ts

```diff
@@ -115,10 +122,18 @@ export function DashboardCustomizePanelProvider({ getService, getPageObject }: F
       });
     }
 
-    public async clickToggleShowCustomTimeRange(enable?: boolean) {
-      log.debug('clickToggleShowCustomTimeRange');
+    public async enableCustomTimeRange() {
```

Comment:
This looks great. One more thing, since it sounds like missing clicks is one cause of flakyness, how about adding a retry

```
const toggle = await testSubjects.find(this.TOGGLE_TIME_RANGE_TEST_SUBJ);
await this.retry.try(async () => {
  if (!(await toggle.isSelected())) {
    await toggle.click();
    if (!(await toggle.isSelected())) {
      throw new Error('Switch click missed');
    }
  }
});
```